src/lib.rs

```diff
@@ -269,6 +269,16 @@ pub mod pin {
             }
         }
 
+        /// Stops the software handling of this pin.
+        ///
+        /// _Note_: In order to control this pin via software PWM again
```

Comment:
This note is a bit redundant since `pwm_stop` consumes `self` anyway. However, it states the intent more clearly and as a new-ish rust user I like this redundancy.